app/scripts/controllers/user-storage/user-storage-controller.test.ts

```diff
@@ -301,6 +301,7 @@ function mockUserStorageMessenger() {
       'MetamaskNotificationsController:disableMetamaskNotifications',
       'MetamaskNotificationsController:selectIsMetamaskNotificationsEnabled',
     ],
+    allowedEvents: [],
```

Comment:
Prior to `@metamask/base-controller` 5.0.0, `allowedEvents` was optional if no events were allowlisted; now this argument is required. See changelog: https://github.com/MetaMask/core/blob/main/packages/base-controller/CHANGELOG.md#changed-2